system-tests/tests/regression/cre/cre_regression_suite_test.go

```diff
@@ -34,7 +37,7 @@ func Test_CRE_V2_Cron_Regression(t *testing.T) {
 	}
 }
 
-func Test_CRE_Suite_V2_HTTP_Regression(t *testing.T) {
+func Test_CRE_V2_HTTP_Regression(t *testing.T) {
```

Comment:
shouldn't we stop changing test names all the time? it makes using trunk.io in any meangifull way close to impossible 🙏